src/common/components/fix-instruction-panel.scss

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../styles/colors.scss';
+
+.insights-fix-instruction-list {
```

Comment:
With this in place, can the corresponding styles be removed from the higher-level scss files where it's repeatedly defined? (`how-to-fix-card-row.scss`, `detailsview.scss`, `injected.scss`, `instance-details.scss`)

Reply:
Yes. Will Do.